app/lib/display-helpers.js

```diff
@@ -11,10 +11,10 @@ function getTypeOfReviewForDisplay (claimData) {
 
 function getSpeciesEligbileNumberForDisplay (claimData) {
   return {
-    beef: 'More than 10 beef cattle ',
-    dairy: 'More than 10 dairy cattle',
-    pigs: 'More than 50 pigs ',
-    sheep: 'More than 20 sheep '
+    beef: '11 beef cattle or more ',
```

Comment:
Which ticket covers this content change?

Reply:
https://fcp-ahwr-prototype.herokuapp.com/v10/defraid-claim/defraid-1/check-answers

https://dev.azure.com/defragovuk/DEFRA-FFC/_workitems/edit/169090

One ticket per species